code/src/App.js

```diff
@@ -1,12 +1,21 @@
 import React from 'react'
 import data from './data.json'
-
-console.log(data)
+import playlist_data from './stretch-goal.json'
+import Header from './components/Header'
+import Album from './components/Album'
+import Playlists from './components/Playlists'
```

Comment:
There is a convention that says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, style.css should be separated as well and located below local dependencies.